java/build-jar-multiplatform.sh

```diff
@@ -131,6 +134,20 @@ prepare_native() {
   done
 }
 
+check_native() {
+  for os in 'darwin' 'linux'; do
+    native_dirs=()
+    native_dirs[0]+="$WORKSPACE_DIR/java/runtime/native_dependencies/native/$os"
+    native_dirs[0]+="$WORKSPACE_DIR/streaming/java/streaming-runtime/native_dependencies/native/$os"
```

Comment:
You're trying to append to the array, right? Try this way:
```suggestion
    native_dirs+=("$WORKSPACE_DIR/java/runtime/native_dependencies/native/$os")
    native_dirs+=("$WORKSPACE_DIR/streaming/java/streaming-runtime/native_dependencies/native/$os")
```

Reply:
Thanks for catching this, my mistake